community/helm/charts/incubator/hasura-graphql-engine/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+1. Get the application URL by running these commands:
```

Comment:
Is this file required or was just internal notes?

Reply:
This is actually...the default contents from `helm create`...